src/app/shared/constants/assessmentTypes.ts

```diff
@@ -0,0 +1,21 @@
+import { UtilityType, UtilityTypes } from "./utilityTypes";
+
+export type AssessmentType = "Pump" | "Fan" | "Process heating" | "Steam" | "Compressed Air" | "Water" | "Treasure Hunt" | "Other";
+export const assessmentTypes: Array<AssessmentType> = ["Pump",  "Fan", "Process heating", "Steam", "Compressed Air", "Water", "Treasure Hunt", "Other"];
+
+// Define the ONE - MANY relationship between Assessment Type and Utility Type
+export interface assessmentOption {
```

Comment:
For consistency can we have the constants and types defined starting with capital letters "AssessmentOption" "AssessmentTypes" "AssessmentOptions" please.

Variables start lower case.